docs/src/config/ini-config.adoc

```diff
@@ -1295,19 +1295,19 @@ image::images/encoder-scale.png[align="center"]
 
 The _<num>_ specifies the spindle number 0 ... (num_spindles-1) +
 The value of _num_spindles_ is set by `[TRAJ]SPINDLES=` . +
-By default maximum velocity of the spindle in forward and reverse is approximately
-  2147483000 RPM. +
+By default maximum velocity of the spindle in forward and reverse is approximately 2147483000 RPM. +
 By default minimum velocity of the spindle in forward and reverse is 0 RPM. +
 By default the increment is 100 RPM. +
 You change these default by setting the following INI variables:
 
 [NOTE]
-These settings are for the motion controller component. Control screens can limit
- these settings further. 
+These settings are for the motion controller component.
+Control screens can limit these settings further. 
 
 * `MAX_FORWARD_VELOCITY = 20000`
   The maximum spindle speed (in rpm) for the specified spindle. Optional.
-  This will also set MIN_FORWARD_VELOCITY to the negative value unless overridden.
+  This will also set MIN_FORWARD_VELOCITY to the negative value unless overridden
+  by MAX_REVERSE_VELOCITY.
```

Comment:
No, `MAX_REVERSE_VELOCITY` was correct. But it seems that it was wrong in the docs. So correct would be this i think:
```suggestion
  This will also set `MAX_REVERSE_VELOCITY` to the negative value unless overridden
  by MAX_REVERSE_VELOCITY.
```
But then the addendum "  by MAX_REVERSE_VELOCITY" doesn't make sense.

Reply:
What I thought this meant is that you only set the forward direction (the one typically used for cutting as I interpret it) and the reverse direction is auto-set with it to the same value unless another value is given. The transition from MIN to MAX in the same paragraph is then to avoid.

Reply:
Re-read it all again and agree that we have spotted a problem with the documentation. I think [e6384fe](https://github.com/LinuxCNC/linuxcnc/pull/2775/commits/e6384fe771061ce8773e01ed32b511ed6c3eef0f) now gets it right. Shall I squash the two?

Reply:
squashed.

Reply:
Yeah nice, thanks!